charts/ai-software-templates/chatbot/templates/service.yaml

```diff
@@ -3,9 +3,8 @@ kind: Service
 metadata: 
   labels: 
     app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/name: {{ .Release.Name }} 
```

Comment:
nit: think you just added a space at the end here

Reply:
Unfortunately is imported from https://github.com/redhat-ai-dev/ai-lab-app/blob/main/templates/http/base/service.yaml#L15C1-L15C49. I think once we add lint checks in ai-lab-app those issues will be gone from this repo too.

Reply:
gotcha ok